.github/workflows/openssl.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: OpenSSL master
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build:
+    name: CI with openssl master 
+    runs-on: ubuntu-22.04
+    container: fedora:latest
+    steps:
+      - name: Get Date for DNF cache entry
+        id: get-date-dnf
+        run: |
+          echo "date=$(/bin/date -u "+%Y%V")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: DNF cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            /var/cache/libdnf5
+          key: ${{ runner.os }}-dnf-${{ steps.get-date.outputs.date }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-dnf-${{ steps.get-date-dnf.outputs.date }}
```

Reply:
The pipeline looks like it works to restore the cache, but it fails to store the cache in `Post DNF cache` stage::
```
Cache saved with key: Linux-dnf-
```